src/providers/ldap/ldap_child.c

```diff
@@ -61,7 +61,6 @@ static void sig_term_handler(int sig)
 static krb5_context krb5_error_ctx;
 #define LDAP_CHILD_DEBUG(level, error) KRB5_DEBUG(level, krb5_error_ctx, error)
 
-static const char *__ldap_child_krb5_error_msg;
```

Comment:
Definition of macro KRB5_SYSLOG is not removed. If sb would try to use it it would fail due to missing definition of variable `__ldap_child_krb5_error_msg`